### ISSUE TITLE
Mandantenbuchhaltung

### ISSUE THREAD

**Issue by stm9x9** [jlawyerorg/j-lawyer-org#527]
Angenehm wäre es, wenn man die offenen / bezahlten Rechnungen an die Mandanten in j-l verwalten könnte:

Die mit den Berechnungsplugins erstellten Rechnungen sollten der Akte zugeordnet werden (also nicht nur als Dokument), man sollte dann schon auf der ersten Seite sehen, ob / wieviel noch offen ist in dem Mandat (zb verabredet der Anwalt ja Vorschuß und daß er bestimmte Verfahrensschritte erst nach Geld-Eingang vornimmt) und dann auf einem neuen Reiter genau: welche Rechnungen gibt es, wann wurde was gezahlt (idealerweise auch auf welche von mehreren Rechnungen wurde gezahlt).

Das sollte dann auch ein Kriterium für die Suche sein. Also: zeige alle Akten mit noch ausstehenden Geldern. zb.

Und wenn man dann diese Buchhaltung noch mit zb csv-Dateien von Banken füttern könnte und die dortigen Einzahlungen, soweit es geht, den Rechnungen schon zugeordnet würden ...

**Comment by stm9x9** [comment]
das hier ist eine schöne Übersicht, wie es zb Advoware gelöst hat: https://onlinehilfe.advoware.de/index.html#!Documents/mandantenbuchhaltung1.html

**Comment by mal02LE** [comment]
Fakturama ist ein offenes Programm zur Buchhaltung, weil ich eine sinnvolle Erfassung nur mit dem Datenblatt (und damit nicht digital) machen kann, läuft das neben Jl-awyer. Schön wäre tatsächlich eine Übersicht "bis wo ist abgerechnet",  "ist ein Vorschuss gezahlt" etc, oder eine Einbindung von (offenen) Buchhaltungsprogrammen

**Comment by j-dimension** [comment]
Tipps von @RA-crm :

https://github.com/willuhn/syntax / https://www.willuhn.de/products/syntax/license.php
https://www.jes-eur.de/?Startseite

siehe Mail RA-crm 14.5.2020

**Comment by stm9x9** [comment]
Ich bin mir nicht sicher, daß eine normale EÜR hier ausreicht. Eine Mandanten- und Mandatsbezogene offene-Posten-Buchhaltung ist das notwendige IMHO.

**Comment by mal02LE** [comment]
Das sehe ich auch so, ich habe gerade wieder eine Email beantwortet, die
aufs Minutenkonto geht, aber keine einzelne Rechnung erzeugt.
Ein Zeit-tool, zur Erfassung der geöffneten, bearbeiteten Akte kann auch
sinnvoll - in diesem Zusammenhang - sein.

Am 14.05.20 um 23:02 schrieb stm9x9:

**Comment by PBaumfalk** [comment]
Gleichzeitig könnte man auch ein Forderungskonto einbauen...

Mit freundlichen Grüßen / Kindly Regards

Patrick Baumfalk
Rechtsanwalt / Lawyer


Mobil:    +49 1525 53 04 194
E-Mail:   patrick.baumfalk@kanzlei-baumfalk.de


Baumfalk - Rechtsanwaltskanzlei
Hauptstr. 195
50169 Kerpen

Tel.: +49 2273 40 68 504
Fax: +49 2273 59 03 565

E-Mail: info@kanzlei-baumfalk.de
Web: www.kanzlei-baumfalk.de

Sent by Mobile

**Comment by Dickmaulruessler** [comment]
Ja, dieser gesamte Komplex ist in der Tat das was fehlt - und in einer Minimalvariante eigentlich auch unverzichtbar ist. 

Ich könnte mir aber sehr gut eine stark abgeschichtete, die Ressourcen unseres _"Chefs"_ (Jens) schonende Entwicklung vorstellen. Es muss nicht gleich eine ganze "Fibu", ein automatischer Kontoabgleich oder eine Schnittstelle sein... 

Mir wäre z.B. schon geholfen, wenn man 

- einen Rechnungsbetrag aktenbezogen manuell eintragen kann

- eine allgemeine aktenübergreifende Übersicht nach "noch ausstehend / bezahlt" (mit manueller Abhakmöglichkeit oder manueller Notiz-Funktion) und 

- eine Monats- Quartals- und Jahresübersicht mit Summenbildung hätte. 

Das sind zwar immerhin schon drei Punkte (vielleicht könnte man für den Start den letzten Punkt auch noch streichen...), aber mein Anliegen war es, die Wünsche in diesem wichtigen Bereich klein und überschaubar zu halten. Wir dürfen Jens auch nicht mit Wünschen, die er zeitlich gar nicht schaffen kann, zuschütten ... Thanks, by the way, Jens!

**Comment by j-dimension** [comment]
Danke @Dickmaulruessler , genau so würde ich mir das vorstellen - es muss iterativ vorangehen. Ich tendiere auch dazu, erstmal die aktenbezogenen Beträge in der Akte erfassbar zu machen (samt Status) und einen aktenübergreifenden Überblick zu ermöglichen.

Die eigentliche Fibu wäre darauf aufbauend.

**Comment by RA-crm** [comment]
siehe: https://github.com/jlawyerorg/j-lawyer-org/issues/756

**Comment by iradraconis** [comment]
> Danke @Dickmaulruessler , genau so würde ich mir das vorstellen - es muss iterativ vorangehen. Ich tendiere auch dazu, erstmal die aktenbezogenen Beträge in der Akte erfassbar zu machen (samt Status) und einen aktenübergreifenden Überblick zu ermöglichen.
> 
> Die eigentliche Fibu wäre darauf aufbauend.

Eins der wichtigsten Features, wie ich meine. Ich bin fast ausschließlich als Strafverteidiger tätig und das Thema Vorschuss vor Tätigkeit ist enorm wichtig. Wenn der Mandant anruft und wissen möchte, wie der Stand ist, möchte ich ihm schnell sagen können, warum ich ggfs. noch nichts oder nicht alles gemacht habe.

**Comment by Pit-123** [comment]
Akten-/Mandantenbuchhaltung ist in der Tat eine wesentliche Funktion. Aber auch eine Fibu. Eine externe Fibu einsetzen und parallel dann noch in der j-lawyer-Aktenbuchhaltung buchen, ist leider nicht so verlockend. Besteht nicht die Möglichkeit, eine existierene Fibu einzubinden?

**Comment by stm9x9** [comment]
Jameica mit SynTax wäre da eine Option. Aber ich weiß nicht, ob das notwendig ist.

**Comment by GuidoBo** [comment]
Was uns ehrlich gesagt am allermeisten fehlt ist eine saubere Zinsberechnung für die Forderungen.

**Comment by j-dimension** [comment]
> Was uns ehrlich gesagt am allermeisten fehlt ist eine saubere Zinsberechnung für die Forderungen.

Vielleicht hilft das hier erstmal weiter?
https://github.com/jlawyerorg/j-lawyer-templates/blob/master/templates/Tabelle/LibreOffice/%5Bjphaag%5D-Forderungskonto.ods

**Comment by LazyGuy21No** [comment]
siehe auch #2163 2163
